packages/hardhat-core/src/internal/core/errors-list.ts

```diff
@@ -51,7 +51,7 @@ export const ERRORS = {
       title: "You are not inside a Hardhat project",
       description: `You are trying to run Hardhat outside of a Hardhat project.
 
-You can learn hoy to use Hardhat by reading the [Getting Started guide](./README.md).`,
+You can learn hoy to use Hardhat by reading the [Getting Started guide](../getting-started).`,
```

Comment:
@alcuadrado is this fix ok? Otherwise the link shown in the errors page of the docs is wrong

Reply:
Yep